lib/reporting/cloudwatch_query.rb

```diff
@@ -0,0 +1,57 @@
+module Reporting
+  # Helper for constructing Cloudwatch Insights queries for common patterns
+  class CloudwatchQuery
```

Comment:
[question] i'm having a hard time understanding how `CloudwatchQuery` is being invoked with the query object -- hopefully we can chat about that this afternoon?

Reply:
updated to remove the Query class in 972d7c487de33aba5b51af82099788d656682938